src/components/ZoumCalculator.js

```diff
@@ -1,51 +1,114 @@
+    } else if (value === '+/-') {
+      setDisplay((prev) => prev * -1);
+    } else if (value === '%') {
+      setDisplay((prev) => prev / 100);
+    } else if (value === '=') {
+      const result = eval(display);
+      setDisplay(result);
+    } else {
+      setDisplay((prev) => prev + value);
+    }
+  };
 
-export function Affiche() {
   return (
     <div>
+      <div className="affichage">{display}</div>
       <div>
-        <button type="button">AC</button>
-        <button type="button">+/-</button>
-        <button type="button">%</button>
-        <button className="ColorBtn" type="button">/</button>
+        <button type="button" onClick={() => handleClick('AC')}>
+          AC
+        </button>
+        <button type="button" onClick={() => handleClick('+/-')}>
+          +/-
+        </button>
+        <button type="button" onClick={() => handleClick('%')}>
+          %
+        </button>
+        <button
+          className="ColorBtn"
+          type="button"
+          onClick={() => handleClick('/')}
+        >
+          ÷
+        </button>
       </div>
       <div>
-        <button type="button">7</button>
-        <button type="button">8</button>
-        <button type="button">9</button>
-        <button className="ColorBtn" type="button">x</button>
+        <button type="button" onClick={() => handleClick('7')}>
+          7
+        </button>
+        <button type="button" onClick={() => handleClick('8')}>
+          8
+        </button>
+        <button type="button" onClick={() => handleClick('9')}>
+          9
+        </button>
+        <button
+          className="ColorBtn"
+          type="button"
+          onClick={() => handleClick('*')}
+        >
+          x
+        </button>
       </div>
       <div>
-        <button type="button">4</button>
-        <button type="button">5</button>
-        <button type="button">6</button>
-        <button className="ColorBtn" type="button">-</button>
+        <button type="button" onClick={() => handleClick('4')}>
+          4
+        </button>
+        <button type="button" onClick={() => handleClick('5')}>
+          5
+        </button>
+        <button type="button" onClick={() => handleClick('6')}>
+          6
+        </button>
+        <button
+          className="ColorBtn"
+          type="button"
+          onClick={() => handleClick('-')}
+        >
+          -
+        </button>
       </div>
       <div>
-        <button type="button">1</button>
-        <button type="button">2</button>
-        <button type="button">3</button>
-        <button className="ColorBtn" type="button">+</button>
+        <button type="button" onClick={() => handleClick('1')}>
+          1
+        </button>
+        <button type="button" onClick={() => handleClick('2')}>
+          2
+        </button>
+        <button type="button" onClick={() => handleClick('3')}>
+          3
+        </button>
+        <button
+          className="ColorBtn"
+          type="button"
```

Comment:
- [ ] Please make sure to use the given logic files to implement the logic, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_events.md#:~:text=Create%20a%20new,your%20logic/%20directory) requirement.